lib/reverse_sentence.rb

```diff
@@ -57,89 +59,8 @@ def reverse_words(sentence)
   end
 end
 
+def reverse_sentence(my_sentence)
   reversed = string_reverse(my_sentence)
```

Comment:
Since you are doing all the string manipulations in place, you don't need to return anything or assign the returned value to anything. reverse_sentence could simply be:
```ruby
   string_reverse(my_sentence)
   reverse_words(my_sentence)
   return
```